payment/src/main/java/org/killbill/billing/payment/core/PaymentRefresher.java

```diff
@@ -311,17 +304,17 @@ public Payment apply(final PaymentModelDao paymentModelDao) {
                                                if (!withPluginInfo) {
                                                    pluginApi = null;
                                                } else {
-                                                   if (paymentMethodIdToPaymentPluginApi.get(paymentModelDao.getPaymentMethodId()) == null) {
```

Comment:
Isn't the `null` check necessary to go through the `if` the first time?

Reply:
Yeah you're right. Sorry.